src/services/email.service.js

```diff
@@ -0,0 +1,59 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: process.env.SMTP_SECURE === true,
```

Comment:
The `secure` option is being set using `process.env.SMTP_SECURE === true`, which will always evaluate to `false` since `process.env.SMTP_SECURE` is a string. Consider using `process.env.SMTP_SECURE === 'true'` to properly evaluate the environment variable as a boolean.